.github/ISSUE_TEMPLATE/suggest-a-new-rule.md

```diff
@@ -0,0 +1,22 @@
+---
+name: New rule suggestion
+about: Suggest a new rule
+title: ''
+labels: 'new-rule'
+assignees: ''
+
+---
+
+<!-- Please refer to https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md -->
```

Comment:
Can this be a hyperlink with relative path? I am not sure if following will work or not.
```suggestion
<!-- Please refer to [GuidelinesForNewRules.md](.//..//..//GuidelinesForNewRules.md) -->
```

Reply:
@mavasani, This won't be rendered as a link, it will appear text as it is, and it will be hard for issue creation to get to the file easily.

Reply:
Ah, I am not aware how this works for issue templates. @Evangelink probably good to revert this change to switch back to https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md

Reply:
reverted